releases/2022-11-25-releasenotes.md

```diff
@@ -0,0 +1,154 @@
+---
+draft: false
+title: "Handle errors the GraphQL way and more results to your query"
+description: "Vendia Share updated the response status code and allowing up to 1000 query results"
+date: '2022-11-25'
+authors:
+  - Li Dai
+tags:
+- Improvements
+- Developer Experience
+- GraphQL
+---
+
+Vendia Engineers worked tirelessly to improve our product and ensure that we follow and set best practices for industrial standards. Here are the newest updates!
```

Comment:
Recommend changing to:
...to improve our product and leverage industry standards.

Reply:
accepted